GameServer/Logic/GameObjects/Spell.cs

```diff
@@ -205,18 +205,17 @@ public Spell(Champion owner, string spellName, byte slot)
             _targetType =
                 (float)Math.Floor(RAFManager.GetFloatValue(data, "Values", "SpellData", "TargettingType") + 0.5f);
 
-            JObject missileData;
-            if (!RAFManager.ReadMissileData(spellName, out missileData))
+            if (!RAFManager.ReadSpellData(spellName + "Missile", out data))
```

Comment:
...didn't you implement a function that does exactly this? `RAFManager.ReadMissileData()`?